lib/crowbar/client/config.rb

```diff
@@ -162,11 +162,9 @@ def default_anonymous
       #
       def default_apiversion
         if ENV["CROWBAR_APIVERSION"].present?
-          [
-            1.0, 2.0
-          ].include? ENV["CROWBAR_APIVERSION"]
+          ENV["CROWBAR_APIVERSION"].to_f
```

Comment:
I guess that was just for the looks (style of the code), as we did it in crowbar-init and crowbar-core as well. in the api constraint (where we check the version) it is converted to a string.